tests/unit/test-util/test.adb.js

```diff
@@ -622,6 +632,35 @@ describe('utils/adb', () => {
        });
   });
 
+
```

Comment:
Nit, remove this unnecessary empty line (there was already one at line 634 to separate the two statements).